### PR TITLE
Update Azure file feature gates for 1.20 beta

### PR DESCRIPTION
**Pull request by andyzhangx**
Docs for https://github.com/kubernetes/enhancements/issues/1885

/assign @msau42

### CODE REVIEW

**Comment by andyzhangx** [comment]
related k/k PR: https://github.com/kubernetes/kubernetes/pull/96293

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -70,7 +70,8 @@ different Kubernetes components.
 | `CSIMigrationAzureDisk` | `false` | Alpha | 1.15 | 1.18 |
 | `CSIMigrationAzureDisk` | `false` | Beta | 1.19 | |
 | `CSIMigrationAzureDiskComplete` | `false` | Alpha | 1.17 | |
-| `CSIMigrationAzureFile` | `false` | Alpha | 1.15 | |
+| `CSIMigrationAzureFile` | `false` | Alpha | 1.15 | 1.19 |
+| `CSIMigrationAzureFile` | `false` | Beta | 1.20 | |
 | `CSIMigrationAzureFileComplete` | `false` | Alpha | 1.17 | |
```
Comment (msau42) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:75]: Oh I forgot about the complete feature gate. Shall we move those to beta too?

**Comment by annajung** [comment]
Hey @andyzhangx  please change the base to `dev-1.20` branch

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (andyzhangx) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:75]: what does `CSIMigrationAzureFileComplete` really mean? is that just a placeholder flag? I see that flag in other cloud provider is also `Alpha` now?

**Comment by andyzhangx** [comment]
> Hey @andyzhangx please change the base to `dev-1.20` branch

already created a new PR: https://github.com/kubernetes/website/pull/24940
@annajung shall we also merge this PR?

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (msau42) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:75]: The complete feature gate, when turned on, will cause the intree plugin to not be registered with kube-controller-manager and kubelet. https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/app/plugins_providers.go#L38

Comment (msau42) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:75]: I think we forgot to move the other drivers feature gates to beta. I plan to clean up the other drivers in 1.21

Comment (andyzhangx) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:75]: let's clean up that complete flag in all these drivers together in 1.21, thanks.

**Comment by annajung** [comment]
Hi @andyzhangx, no need to create a new PR. I believe you have an `edit` option on the top right side and once you click on it, you should be able to change the base to `master`.

**Comment by andyzhangx** [comment]
> Hi @andyzhangx, no need to create a new PR. I believe you have an `edit` option on the top right side and once you click on it, you should be able to change the base from `master` to `dev-1.20`.

these two branches has too many diff, I would like to create new PR. And what about `master` branch, do we still maintain this branch?

**Comment by annajung** [comment]
Sure, if easier, you can close out this PR and keep https://github.com/kubernetes/website/pull/24940 
`master` is being maintained but all work related to the release needs to be part of `dev-1.20` branch. The `dev-1.20` branch will be merged to `master` during release day.

**Pull request closed without merging**